### ISSUE TITLE
Final Peer Review_hc936

### ISSUE THREAD

**Issue by HanyinCao** [Archerhaha/LOLGameWinPrediction#12]
Overall, this team did a great job. Video game is a popular issue, especially among young people. This research can help the players make better strategies to win more games. In details, Firstly, this team tried various loss functions and regularizers and made combination among them. The comparison of these models not only help to choose the best model, but also help the readers get deeper understanding about the difference of various loss function and regularizers. Secondly, this research utilized many types of graphs to describe the team's wining probability under different condition. It is of great help for getting general idea about the data. 

However, I think this project can be improved in some aspects. Firstly, the dataset is too "clean". There is no missing value, no outlier, making the dataset seems to be too ideal. In this case, many skills of data processing cannot be used. Secondly, in order to evaluate the predicting accuracy of different model, the dataset should be divided into three parts rather than two parts. They are training set, validation set and testing set. After getting the optimal parameters, the predicting accuracy of testing set really matters. Thirdly, when conducted research on the importance of different features, I don't think the absolute value of coefficient is a good standard. Because some variables are boolean while others are integer, the scales of the data are different. In this case, comparing the value of the coefficients is not a good way to judge the importance of features.
